bin/luster.js

```diff
@@ -2,6 +2,7 @@
 const /** @type {ClusterProcess} */
     luster = require('../lib/luster'),
     path = require('path'),
-    configFilePath = path.resolve(process.cwd(), process.argv[2] || 'luster.conf');
+    // config path is right after this script in process.argv
+    configFilePath = path.resolve(process.cwd(), process.argv[process.argv.indexOf(__filename) + 1] || 'luster.conf');
```

Comment:
`__filename` is an absolute path, while the path of an entrypoint passed in the args may be relative. Argument needs to be resolved into an absolute path before comparison.